server.js

```diff
@@ -4,9 +4,18 @@ const connectDB = require('./config/db')
 const app = express()
 
 connectDB()
-
 app.use(express.json({ extended: false }))
 
+app.use((req, res, next) => {
+  res.header('Access-Control-Allow-Origin', '*')
+  res.header('Access-Control-Allow-Headers', 'Origin,X-Requested-With, Content-Type, Accept, Authorization')
+  if (req.method === 'OPTIONS') {
+    res.header('Access-Control-Allow-Methods', 'PUT, POST, PATCH, DELETE, GET')
+    return res.status(200).json({})
+  }
+  next()
+})
```

Comment:
@artlemus You can use the cors package for this: https://expressjs.com/en/resources/middleware/cors.html